test/e2etest/bamboo_test.go

```diff
@@ -289,3 +289,12 @@ func GetPageContent(t *testing.T, url string) []byte {
 	assert.NoError(t, err, "Error reading response body")
 	return content
 }
+
+//PostPageContent performs a post request the given content using the given url
+func PostPageContent(t *testing.T, url string, contentType string, body io.Reader) {
```

Comment:
1. Can we change function name to something more precise? Post page content doesn't feel quite right to me. 
A good function name should be self-explanatory, and theoretically should not need any comment.  

2. If none of the caller uses `body`, maybe we can omit it here as an input parameter? 

3. Noticing the same for function `GetPageContent`, any reason we're making them public? Looks like they're only been used within the package.

Reply:
I followed existing GetPageContent and you are right. Those are not needed to be public and I renamed the method to `sendPostRequest`. 

We still need to keep the body as this is part of post request convention. We may not using them for our cases but good to have just in case if we want to use in future.